services/movie_session.py

```diff
@@ -41,4 +41,14 @@ def update_movie_session(
 
 
 def delete_movie_session_by_id(session_id: int) -> None:
-    MovieSession.objects.get(id=session_id).delete()
+    try:
+        MovieSession.objects.get(id=session_id).delete()
+    except MovieSession.DoesNotExist:
+        print(f"Movie session with id {session_id} does not exist.")
+
+
+def get_taken_seats(movie_session_id: int) -> list:
```

Comment:
There's a typo in the function name `get_taken_seats` in the previous review. Ensure the function name is correctly used throughout the codebase to avoid confusion.

Reply:
Agree